keras/engine/sequential.py

```diff
@@ -46,44 +46,44 @@ class Sequential(functional.Functional):
 
   Examples:
 
-  >>> # Optionally, the first layer can receive an `input_shape` argument:
-  >>> model = tf.keras.Sequential()
-  >>> model.add(tf.keras.layers.Dense(8, input_shape=(16,)))
-  >>> # Afterwards, we do automatic shape inference:
-  >>> model.add(tf.keras.layers.Dense(4))
-
-  >>> # This is identical to the following:
-  >>> model = tf.keras.Sequential()
-  >>> model.add(tf.keras.Input(shape=(16,)))
-  >>> model.add(tf.keras.layers.Dense(8))
-
-  >>> # Note that you can also omit the `input_shape` argument.
-  >>> # In that case the model doesn't have any weights until the first call
-  >>> # to a training/evaluation method (since it isn't yet built):
-  >>> model = tf.keras.Sequential()
-  >>> model.add(tf.keras.layers.Dense(8))
-  >>> model.add(tf.keras.layers.Dense(4))
-  >>> # model.weights not created yet
-
-  >>> # Whereas if you specify the input shape, the model gets built
-  >>> # continuously as you are adding layers:
-  >>> model = tf.keras.Sequential()
-  >>> model.add(tf.keras.layers.Dense(8, input_shape=(16,)))
-  >>> model.add(tf.keras.layers.Dense(4))
-  >>> len(model.weights)
+  ```python
+  # Optionally, the first layer can receive an `input_shape` argument:
+  model = tf.keras.Sequential()
+  model.add(tf.keras.layers.Dense(8, input_shape=(16,)))
+  # Afterwards, we do automatic shape inference:
+  model.add(tf.keras.layers.Dense(4))
+
+  # This is identical to the following:
+  model = tf.keras.Sequential()
+  model.add(tf.keras.Input(shape=(16,)))
+  model.add(tf.keras.layers.Dense(8))
+
+  # Note that you can also omit the `input_shape` argument.
+  # In that case the model doesn't have any weights until the first call
+  # to a training/evaluation method (since it isn't yet built):
+  model = tf.keras.Sequential()
+  model.add(tf.keras.layers.Dense(8))
+  model.add(tf.keras.layers.Dense(4))
+  # model.weights not created yet
+
+  # Whereas if you specify the input shape, the model gets built
+  # continuously as you are adding layers:
+  model = tf.keras.Sequential()
+  model.add(tf.keras.layers.Dense(8, input_shape=(16,)))
+  model.add(tf.keras.layers.Dense(4))
+  len(model.weights)
   4
```

Comment:
IMO the whole thing does become a lot more readable as code blocks like this. But the output of the code ("4" here) should be moved to a comment, like `# Returns "4"`

Reply:
Yup that change would be good. Also, there are lots of codes where these `>>>` exists so should I make changes for them all one by one?